data-otservbr-global/scripts/creaturescripts/quests/ferumbras_ascendant/ferumbras_soul_splinter.lua

```diff
@@ -1,10 +1,15 @@
-local ferumbrasSoulSplinter = CreatureEvent("FerumbrasSoulSplinter")
+local ferumbrasSoulSplinter = CreatureEvent("FerumbrasSoulSplinterDeath")
+
 function ferumbrasSoulSplinter.onDeath(creature, corpse, lasthitkiller, mostdamagekiller, lasthitunjustified, mostdamageunjustified)
-	local monster = Game.createMonster("ferumbras essence", creature:getPosition(), true, true)
-	if not monster then
-		return true
-	end
+	Game.createMonster("ferumbras essence", creature:getPosition(), true, true)
```

Comment:
Aconselho manter a verificação e adicionar um log.
```suggestion
	local monster = Game.createMonster("ferumbras essence", creature:getPosition(), true, true)
	if not monster then
		logger.error("[ferumbrasSoulSplinter.onDeath] cannot create monster on position {}", creature:getPosition():toString())
		return true
	end
```